packages/atomic/src/themes/coveo.css

```diff
@@ -31,6 +31,11 @@
   --atomic-font-normal: 300;
   --atomic-font-bold: 500;
 
+  /* Text size */
+  --atomic-text-base: 0.875rem; /* 14px */
```

Comment:
Super useful to easily switch the font size throughout the interface, 12px can be too small for some, even if its in our designs

Reply:
There will probably be heading sizes too for the Templates